extensions/widgets/svgpath/svgpath.lcb

```diff
@@ -366,7 +366,10 @@ public handler OnPaint()
 	else
 		set the paint of this canvas to solid paint with mHiliteColor
 	end if
-	
+
+	-- FIXME Make this configurable via a property
+	set the fill rule of this canvas to "non-zero"
```

Comment:
If we are ensuring the default winding rule is non-zero, then do we need to set things explicitly in the widget? Or this so we can make it configurable in the future...